museum/schema/dumptomysql

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# http://nialldonegan.me/2007/03/10/converting-microsoft-access-mdb-into-csv-or-mysql-in-linux/
+
+# stop on errors and undefined variables
+set -e
+set -u
+
+export MDBOPTS="use_index;debug_all"
+export MDB_COPY_OLE
+
+SOURCE="$1"
+TARGET="$2"
+MYSQL_OPTIONS=""
+if [ $# -gt 2 ]; then
+	MYSQL_OPTIONS="$3"
+fi
+
+echo "Got the following arguments:"
+echo " + SOURCE = $SOURCE"
+echo " + TARGET = $TARGET"
+echo " + MYSQL_OPTIONS = $MYSQL_OPTIONS"
+
+echo "Creating fresh database $TARGET"
+SQL="DROP DATABASE IF EXISTS \`$TARGET\`; CREATE DATABASE \`$TARGET\`;"
+mysql ${MYSQL_OPTIONS} -e "$SQL"
+TEMPDIR=`mktemp -d`
+TARGET_SCHEMA="$TARGET"
+TARGET="$TEMPDIR/$TARGET"
+echo "Exporting MDB schema to $TARGET.mdbschema.sql"
+mdb-schema --no-not-null --no-not-empty "$SOURCE" mysql > "$TARGET.mdbschema.sql"
+
+echo "Cleaning MDB schema"
+cat "$TARGET.mdbschema.sql" |
+grep -v "COMMENT ON COLUMN" | #cos comments break mysql
+sed -e 's/\(CREATE TABLE.*\)\//\1-/g' | #and can't import from files with a slash in their  name
+sed -e 's/text (255)/text/g'| #these are all memo
+sed -e 's/varchar (255)/blob/g'| #these are all ole fields
```

Comment:
This seems like a strange mapping, but as discussed, it does not apply to this database so can be removed.